external/curl/Makefile

```diff
@@ -51,6 +51,7 @@ OBJS		= $(AOBJS) $(COBJS)
 # Make Own Archive for CURL
 BIN		= ../libexternal$(LIBEXT)
 
+CFLAGS+= -D__TIZENRT__
```

Comment:
what does `__TIZENRT__`  specifies here?  If possible please use appropriate naming convention.

Reply:
@lokeshbv 
"__TIZENRT__" is used across multiple modules in TizenRT. IoTivity, LibTUV, IoT JS for instance.